### PR TITLE
Grid Tabbing

### PR DESCRIPTION
**Pull request by etokareva**

### CODE REVIEW

**Review by vkorolev — APPROVED** [review]

File: src/core/navigation/navigation.js [hunk 1]
```diff
@@ -65,6 +65,36 @@ export default class Navigation {
 					}
 				}
 			}),
+			tabNav: new Command({
```
Comment (klumba12) [inline src/core/navigation/navigation.js:68]: rename to tab

File: src/core/navigation/navigation.js [hunk 2]
```diff
@@ -65,6 +65,36 @@ export default class Navigation {
...
+				shortcut: 'tab',
+				canExecute: () => model.edit().editMode == 'view',
+				execute: () => {
+					const lastCell = model.navigation().column === columnService.lineView(model.view().columns).length - 1;
+					const lastRow = model.navigation().row === model.view().rows.length - 2;
```
Comment (klumba12) [inline src/core/navigation/navigation.js:73]: why -2?

File: src/core/navigation/navigation.js [hunk 3]
```diff
@@ -65,6 +65,36 @@ export default class Navigation {
 					}
 				}
 			}),
+			tabNav: new Command({
+				shortcut: 'tab',
+				canExecute: () => model.edit().editMode == 'view',
+				execute: () => {
+					const lastCell = model.navigation().column === columnService.lineView(model.view().columns).length - 1;
```
Comment (klumba12) [inline src/core/navigation/navigation.js:72]: lastCell -> isLastCell

Comment (klumba12) [inline src/core/navigation/navigation.js:72]: model.navigation() to navigationState = model.navigation()

File: src/core/navigation/navigation.js [hunk 4]
```diff
@@ -65,6 +65,36 @@ export default class Navigation {
...
+					const lastRow = model.navigation().row === model.view().rows.length - 2;
+					if (model.navigation().row == -1 && model.navigation().column == -1) {
```
Comment (klumba12) [inline src/core/navigation/navigation.js:74]: lots of ifs need to refactor?

**Pull request merged**